kyuubi-common/src/main/scala/org/apache/kyuubi/session/SessionManager.scala

```diff
@@ -41,6 +41,8 @@ abstract class SessionManager(name: String) extends CompositeService(name) {
 
   @volatile private var shutdown = false
 
+  def LOG_ROOT: String
```

Comment:
// initialize it in `def initialize(conf: KyuubiConf)`
protected var _operationLogRoot 
def operationLogRoot = _operationLogRoot